src/main/java/org/altbeacon/beacon/BeaconManager.java

```diff
@@ -666,6 +668,36 @@ public static void setDistanceModelUpdateUrl(String url) {
         distanceModelUpdateUrl = url;
     }
 
+    /**
+     * Default class for rssi filter/calculation implementation
+     */
+    protected static Class rssiFilterImplClass = RunningAverageRssiFilter.class;
```

Comment:
Sorry, forget this comment.  This suggestion won't work because the code sets the Class instance, not an Object instance, and only Object instances can use the Interface as a type.  Let's just leave it as-is.